src/java.desktop/share/classes/javax/swing/RepaintManager.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This should be reverted since there are no lanai-specific changes to this file (only the copyright year bump which is therefore not correct).